examples/01_Simulation.ipynb

```diff
@@ -1137,6 +1137,109 @@
     "## Parameters"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "All the previous sections used a model with default settings. This is hardly ever the case. A model will have to be configured to represent the actual system. For example, the BatteryCircuit default parameters are for a 18650 battery tested in the NASA's SHARP lab. If you're using the model for a system other than that one battery, you will need to update the parameters.\n",
```

Comment:
```suggestion
    "All the previous sections used a model with default settings. This is hardly ever the case. A model will have to be configured to represent the actual system. For example, the BatteryCircuit default parameters are for a 18650 battery tested in NASA's SHARP lab. If you're using the model for a system other than that one battery, you will need to update the parameters.\n",
```

Reply:
Do you think this is a good place to include a physical/real example? I'm thinking something like, "in the battery model, the parameters include battery capacity ... ". Just a thought. Parameters feel relatively self-explanatory, so I'm okay without including this too